developer/frameworks/articles/09-info-framework/01-the-info-framework-intro.markdown

```diff
@@ -0,0 +1,81 @@
+# The Info Framework
+
+The Info Framework provides enhances flexibility for retrieving, processing, 
```

Comment:
"enhances" should be "enhanced", although I would rather just remove the word, since it's not necessary.